Explorer/Assets/DCL/Chat/ChatController.cs

```diff
@@ -271,6 +292,18 @@ private async UniTaskVoid SearchAndSetEmojiSuggestionsAsync(string value, Cancel
             emojiSuggestionPanelController.SetPanelVisibility(true);
         }
 
+        private void CloseAllEmojiPanels(InputAction.CallbackContext obj)
```

Comment:
I recommend to revise this method: we already have `UpdateCursorInputSystem` that performs raycasting. You should be consuming the results of that system in the query instead of performing right the same raycast once more.

Reply:
I entirelly removed it because I need to think of a more proper solution considering it was only working on mouse lock but not on a click on different UIs, will do it in another PR